modules/manager/metadata/metadata.go

```diff
@@ -29,7 +44,16 @@ type Info struct {
 	Organization string
 
 	// Flavor is the flavor of the build.
-	Flavor string
+	Flavor BuildFlavor
+}
+
+// UserAgent returns the User-Agent string to use in all HTTP requests made by KGO.
+func (inf Info) UserAgent() string {
+	org := strings.ToLower(inf.Organization)
+	if inf.Flavor == OSSFlavor {
+		return fmt.Sprintf("%s-%s-%s/%s", org, inf.ProjectName, OSSFlavor, inf.Release)
+	}
+	return fmt.Sprintf("%s-%s/%s", org, inf.ProjectName, inf.Release)
```

Comment:
I'd suggest to include
- OS
- arch

Similarly to what Kubernetes in tree components like controller manager do it:

```
kube-controller-manager/v1.30.0 (linux/arm64) kubernetes/7c48c2b/leader-election
```

We can also put the flavor in the UA regardless to it's value

```suggestion
	return fmt.Sprintf("%s-%s/%s (%s/%s) (%s)",
		org, inf.ProjectName, inf.Release, runtime.GOOS, runtime.GOARCH, inf.Flavor,
	)
```

Reply:
I did it like that to be consistent with [what we do in KIC](https://github.com/Kong/kubernetes-ingress-controller/blob/b996cd5c8a978d8a08d69524e1c2ecd0bbbe1821/internal/manager/metadata/metadata.go#L45-L47). More info is better so  I can adjust in both repos to stay consistent. 

In terms of name, I'm leaning towards being consistent with Docker image naming, because this is the most typical way of running it

Reply:
I see. The problem is that we use `ProjectName` here which is bound to GitHub's repo name (not the docker repo name). 

Do you mean by this that you intend to change your initial change?

Reply:
TBH I don't know how to do it in the best way, because we don't have a variable in `Makefile` that maps one to one. So I came up with that construction of the name. On the other hand, I don't want to hardcode anything here. For now names map (repo and Docker image), but in the future whoever knows...